security/uacme/Portfile

```diff
@@ -0,0 +1,55 @@
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                uacme
+version             1.7.4
+revision            0
+categories          security
+license             GPL-3+
+platforms           any
+maintainers         nomaintainer
+description         ACMEv2 client written in plain C with minimal dependencies
+long_description    lightweight client for the RFC8555 ACMEv2 protocol, \
+                    written in plain C with minimal dependencies \
+                    (libcurl and OpenSSL).
+homepage            https://github.com/ndilieto/uacme
+
+github.setup        ndilieto uacme 1.7.4 v
+
+checksums           rmd160  5d6bb1001849f28ae5da01913c2ddce9cb20a502 \
+                    sha256  5dd8ecd6411b1b18548c2660b05acd273e0878c60d8762c412aea56e9de2f727 \
+                    size    257908
+
+depends_build       port:pkgconfig \
```

Comment:
This would be better as `depends_build-append`

Reply:
@herbygillot Can you explain what is better with `depends_x-append`?

Reply:
In a Portfile, certain statements like `depends_build`, `depends_libs`, and `patchfiles` are arrays. They are a list of values.

When you import a portgroup at the beginning of your Portfile, it may make some implicit additions to any of these.

So by using `...-append`, you ensure that you are never overwriting anything by mistake.

Reply:
Thanks for the explanation. So should I also use `patchfiles-append` instead?

Reply:
You don't _have_ to, but it's good practice as well.